src/signals/incident-management/containers/IncidentDetail/components/Attachments/styles.ts

```diff
@@ -63,6 +64,12 @@ export const StyledImg = styled.img`
   object-fit: cover;
 `
 
+export const StyledDocument = styled(DocumentText)`
```

Comment:
maybe this needs some subtle changes. There wasn't really a design, just a asc asset.